src/main/resources/testcontainers.properties

```diff
@@ -1 +1,2 @@
+image.substitutor=uk.gov.hmcts.reform.cdapi.util.ContainerImageNameSubstitutor
```

Comment:
property below configures an image name prefix for testcontainers. Is this property and implementation class still needed?

Reply:
removed this